internal/cli/auth/register.go

```diff
@@ -15,20 +15,76 @@
 package auth
 
 import (
+	"context"
+	"errors"
 	"fmt"
+	"os"
 
 	"github.com/mongodb/mongocli/internal/cli/require"
 	"github.com/mongodb/mongocli/internal/config"
+	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
 )
 
+const accountURI = "https://account.mongodb.com/account/register?fromURI=https://account.mongodb.com/account/connect"
```

Comment:
1) query params are optionals, a person who knows this will not type them
2) this is extremely large if I have to type it, consider that one of the use cases of device flow is that the machine I'm using doesn't have access to a browser leaving me to manually type this

Reply:
sure, should we leave only the register page for now? 

to add context: this is for a MVP though, in the end we would have something like what's in the spec -> "https://account.mongodb.com/account/register?source=atlascli orhttps://account.mongodb.com/account/register?utm_source=atlascli